src/api/index.js

```diff
@@ -1,16 +1,16 @@
-import _ from "lodash";
-import axios from "axios";
-import SETTINGS from "../settings";
+// import _ from 'lodash';
```

Comment:
let's go ahead and remove anything that's commented out.

Reply:
Fixed